packages/task/src/browser/task-configurations.spec.ts

```diff
@@ -0,0 +1,174 @@
+/* eslint-disable */
+/********************************************************************************
+ * Copyright (C) 2019 Red Hat, Inc. and others.
```

Comment:
This should reflect your company, I believe in your case `SAP`?

Reply:
fixed

Reply:
fixed